pages/sdk-protocol-kit/guides/safe-signatures.md

```diff
@@ -0,0 +1,848 @@
+# Safe Signatures
+
+Understanding and generating signatures can be challenging. In the **Safe{Core} SDK**, we provide a set of utilities that can assist in using signatures with Safe. In this article, we will explain how signatures work and how to generate them using the `@safe-global/protocol-kit` package.
+
+## Setting up the example Safe Account
+
+Your Safe Account can be configured with various thresholds and different owner types. An owner can be any Ethereum address, such as:
+
+- External Owned Account (EOA)
+- Child Signer Safe (A Safe Account that's an owner of another Safe Account)
+
+When the owner is an EOA, we generate a signature that's different from the signature created using a Safe Account. The Safe Account is a Smart Contract Account, so we need to consider this when collecting the signatures, as the Safe Accounts validate them differently.
```

Comment:
I think you can be more precise here:
```suggestion
When the owner is an EOA, an ECDSA signature is generated. When the signer is a smart account, the exact signature algorithm depends on the signer itself.
```

Reply:
> so we need to consider this when collecting the signatures, as the Safe Accounts validate them differently.

I don't know what considerations are needed when collecting the signatures. There are some when encoding the signatures, for sure